generator/README.md

```diff
@@ -64,9 +64,17 @@ Dev mode is the way to use all new extensions from `master` branch:
 And `che:theia` will use `master` branch for all extensions and plugins, regardless of provided configuration
 
 ### Development life-cycle
-it's easy to check changes. Running yarn watch in a module and then running yarn watch in `examples/assembly` folder and using `yarn run start` in `examples/assembly`
+Che Theia should be built from root directory only. In case of building from subdirectories it will mess up dependencies, don't do it.
 
-### compiling the plugins
+To build whole Che Theia just execute `yarn` command in the root directory.
+If only one module should be built, use `npx run build <module-name>`. For example `npx run build @theia/plugin-ext`.
+
+Also it's easy to check changes. Run `yarn watch` in a module and then execute `yarn watch` from `examples/assembly` folder and run `yarn run start` from the same directory. Make sure, you start watcher in all modules under development.
```

Comment:
I suppose there should be "Run `npx run watch <module-name>` in the root directory" instead of "Run `yarn watch` in a module".

Reply:
Yes, good note, thank you. I just left this one from previous doc writer...